src/js/test/integration/pyodide.test.js

```diff
@@ -11,9 +11,16 @@ describe("Pyodide", () => {
   describe("micropip", () => {
     before(async () => {
       const factory = async () => {
-        await pyodide.loadPackage(["micropip"]);
+        return pyodide.loadPackage(["micropip"]);
```

Comment:
IIRC we are not running the integration tests in the CI right now. Could you confirm that this test passes?

Reply:
Sure, I have checked that the test passes but I had to install an additional package that wasn't in `package.json`, `puppeteer`, and run `conftest.js` manually before the tests. Is there documentation on how to run these tests that im missing?

Reply:
> puppeteer, and run conftest.js manually before the tests. Is there documentation on how to run these tests that im missing? Or should these steps be added in package.json?

I apologize for the difficulty in testing. We haven't used JS tests much so far, so we don't have a lot of documentation on them. As for puppeteer, We've removed it from dependencies (https://github.com/pyodide/pyodide/issues/3332) for now because installing that package affects build times. In the long run, we plan to replace it with something like playwright.